src/tables.rs

```diff
@@ -672,47 +655,47 @@ pub static dav1d_filter_mode_to_y_mode: [u8; 5] = [
 pub static dav1d_intra_mode_context: [u8; N_INTRA_PRED_MODES] =
     [0, 1, 2, 3, 4, 4, 4, 4, 3, 0, 1, 2, 0];
 
-pub static dav1d_wedge_ctx_lut: [u8; N_BS_SIZES] = [
+pub static dav1d_wedge_ctx_lut: [u8; BlockSize::COUNT] = [
     0, 0, 0, 0, 0, 0, 0, 6, 5, 8, 0, 4, 3, 2, 0, 7, 1, 0, 0, 0, 0, 0,
 ];
 
 pub const cfl_allowed_mask: c_uint = 0
-    | 1 << BS_32x32
-    | 1 << BS_32x16
-    | 1 << BS_32x8
-    | 1 << BS_16x32
-    | 1 << BS_16x16
-    | 1 << BS_16x8
-    | 1 << BS_16x4
-    | 1 << BS_8x32
-    | 1 << BS_8x16
-    | 1 << BS_8x8
-    | 1 << BS_8x4
-    | 1 << BS_4x16
-    | 1 << BS_4x8
-    | 1 << BS_4x4
+    | 1 << BlockSize::Bs32x32 as u8
```

Comment:
I think for readability it'd be better to glob import the `BlockSize` variants at the top of the and then use the variants directly, like what you did for `dav1d_block_sizes`. Same applies to `wedge_allowed_mask` and `interintra_allowed_mask`.

Reply:
did that now, but it does format to something slightly uglier

Reply:
Could you stick a `#[rustfmt::skip]` on there? If not I leave it up to you to decide which approach you like better since it's just a style issue.

Reply:
nah it's allright, it's fine this way

Reply:
It doesn't really matter whichever way it is, but that's why I didn't suggest using `use BlockSize::*` for those.  We could also put all of the `Bs`s in an array (without the `as u8`s) and then do a `const_for!` loop to `|` them together.  It's fine as is, though, so I'll merge it.